docs/conf.py

```diff
@@ -12,14 +12,14 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
-# import os
-# import sys
-# sys.path.insert(0, os.path.abspath('.'))
+import os
+import sys
+sys.path.insert(0, os.path.abspath('.'))
 
 
 # -- Project information -----------------------------------------------------
 
-project = 'Property calculation toolkit from the Open Forcefield Consortium.'
+project = 'Property Estimator'
```

Comment:
That's weird. I've started an issue on the cookiecutter repo to change this. https://github.com/MolSSI/cookiecutter-cms/issues/64